board/kent/gfxdemo/MAINTAINERS

```diff
@@ -0,0 +1,6 @@
+KENT GFXDEMO BOARD
+M:	Sam Birch <sbirch@innovative-electronics.com>
```

Comment:
Do you want to use your profirmserv.com address for this?

Reply:
I'm actually kinda unclear on my "identity" for the purposes of this file and also for the copyright statements and email addresses there. I've been putting IELLC and using my IELLC address, but since (I assume) this is work for hire, it should probably be ©Kent and, sure, I should like to use sam@profirmserv.com address.

Reply:
Also I should mention that I'm pretty skeptical that devicetree files are eligible for copyright protection, but here we are.

Reply:
I have normalized copyright and license statements.

Reply:
OK, be all that as it may, I don't think this particular line of code is a copyright statement. It's a maintainer pointer.